elasticsearch/src/test/java/org/hibernate/search/elasticsearch/test/ElasticSearchFlushIT.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.elasticsearch.test;
+
+import java.io.Serializable;
+import java.util.Locale;
+
+import org.hibernate.search.annotations.Field;
+import org.hibernate.search.annotations.Indexed;
+import org.hibernate.search.backend.FlushLuceneWork;
+import org.hibernate.search.backend.LuceneWork;
+import org.hibernate.search.bridge.util.impl.ContextualExceptionBridgeHelper;
+import org.hibernate.search.cfg.Environment;
+import org.hibernate.search.elasticsearch.cfg.ElasticsearchEnvironment;
+import org.hibernate.search.elasticsearch.cfg.IndexSchemaManagementStrategy;
+import org.hibernate.search.elasticsearch.client.impl.JestClient;
+import org.hibernate.search.engine.integration.impl.ExtendedSearchIntegrator;
+import org.hibernate.search.engine.service.spi.ServiceManager;
+import org.hibernate.search.engine.service.spi.ServiceReference;
+import org.hibernate.search.engine.spi.DocumentBuilderIndexedEntity;
+import org.hibernate.search.engine.spi.EntityIndexBinding;
+import org.hibernate.search.indexes.spi.IndexManager;
+import org.hibernate.search.query.engine.spi.HSQuery;
+import org.hibernate.search.spi.DefaultInstanceInitializer;
+import org.hibernate.search.testsupport.TestForIssue;
+import org.hibernate.search.testsupport.junit.SearchFactoryHolder;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+
+import io.searchbox.indices.settings.UpdateSettings;
+import org.apache.lucene.search.MatchAllDocsQuery;
+
+public class ElasticSearchFlushIT {
```

Comment:
Could you please use `Elasticsearch` (no capital "S"), just to be consistent with other tests?